database/index.js

```diff
@@ -0,0 +1,22 @@
+const mysql = require('mysql');
+const mysqlConfig = require('./config.js');
+
+const connection = mysql.createConnection(mysqlConfig);
+
+//create functions to get required info from the db.  see if you can have them coming in descending order by review ID number so the newest is up front.
+const getExpReviews = (expId, callback) => {
+  let query = 'select * from reviews where experience_id = ?';
+  let experience_id = expId;
+  connection.query(query, experience_id, (err, response, body) => {
```

Comment:
Are you doing anything with the "body" parameter that you are passing in?

Reply:
Nothing that I can see right now.  I can remove it.